src/main/index.ts

```diff
@@ -100,6 +101,36 @@ electron.protocol.registerSchemesAsPrivileged([
   },
 ]);
 
+electron.app.once("ready", (_) => {
```

Comment:
We already have once ready in this file, why do we need another?

Reply:
Seperation. Its easier to read separated callbacks instead of something mushed together.